store/actions/ActionTypes.js

```diff
@@ -0,0 +1,10 @@
+/** @format */
+
+export default {
+	BLOCK: {
+		FOCUS: 'BLOCK_FOCUS_ACTION',
+		MOVE_UP: 'BLOCK_MOVE_UP_ACTION',
+		MOVE_DOWN: 'BLOCK_MOVE_DOWN_ACTION',
+		DELETE_ACTION: 'BLOCK_DELETE_ACTION',
```

Comment:
This should be `DELETE: 'BLOCK_DELETE_ACTION'` here not `DELETE_ACTION: 'BLOCK_DELETE_ACTION'`

Reply:
Nice catch! Which makes me wonder how the tests pass :). Will look into it!

Reply:
There should be a way to type check this with Flow, like in `block-management/constants.js`

Reply:
> Nice catch! Which makes me wonder how the tests pass :). Will look into it!
The test passes because both are `undefined`

Reply:
OK, added some changes to make sure Actions, ActionTypes, the reduser and the store creation are included in Flow. Pretty sure more code will need the same treament. Will do in separate commits.